.travis.yml

```diff
@@ -82,53 +87,58 @@ matrix:
   allow_failures:
     - python: "nightly"
 
-# before_install:
+before_install:
```

Comment:
I keep flip flopping on switching virtualenv on and off.  I wasn't looking at it closely enough.  I'll fix it.